pom.xml

```diff
@@ -44,6 +44,8 @@
 		<selenium.version>3.6.0</selenium.version>
 		<slf4j.version>1.7.25</slf4j.version>
 		<apache.http.version>4.5.3</apache.http.version>
+        <apache.commons.vsf2>2.0</apache.commons.vsf2>
```

Comment:
I often put versions in the properties because when you have a lot of dependencies, it can be nice to see the versions at one glance. However it's more a 'being used to' then really a practice. 
I will update.